BLSQ/iaso
# POLIO 661: campaign history UI improvements

Poor readibility of the the data coming from logs especially for nested values (rounds, scopes,...)
This PR was made in pair coding with @beygorghor to improve the UI

Related JIRA tickets : [POLIO-661](https://bluesquare.atlassian.net/browse/POLIO-661?atlOrigin=eyJpIjoiOTYwMzJlY2M2YmZlNDUxNjkwMzYyZDIyMGM4ZjQwOTciLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- add `ExpandableItem` component for complex values with nested objects and arrays (rounds, scopes, vaccines, shipments, destructions)
- create a `config.tsx `file to get the key and convert it if necessary with a `getLogValue` function
- centralize logic to render rows with array/objects in `mapLogStructure` function

## How to test

- Go to Campaign
- Choose a campaign
- Edit some fields
- Click on history icon in the edit dialog
- Check through all the log if values are well displayed (ExpandableItem for nested values, links should work, date in the right format)

## Print screen / video

https://user-images.githubusercontent.com/25134301/226388947-b91e23e0-c662-4bbd-8402-c3c8c6ed2259.mov


[POLIO-661]: https://bluesquare.atlassian.net/browse/POLIO-661?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
